gitops/prepare/prepare.go

```diff
@@ -110,8 +114,13 @@ func ResolveBuildBundle(ctx context.Context, localClient, cli client.Client, nam
 
 // Return true when hacbs configmap exists in the namespace
 func IsHACBS(ctx context.Context, cli client.Client, namespace string) bool {
+	var apiBinding = kcpv1alpha1.APIBinding{}
+	err := cli.Get(ctx, types.NamespacedName{Name: HACBSAPIBindingName}, &apiBinding)
+	if err == nil {
+		return true
+	}
 	var configMap = corev1.ConfigMap{}
-	err := cli.Get(ctx, types.NamespacedName{Name: HACBSConfigMapName, Namespace: namespace}, &configMap)
+	err = cli.Get(ctx, types.NamespacedName{Name: HACBSConfigMapName, Namespace: namespace}, &configMap)
```

Comment:
why do we need to check for hacbs configmap? does it mean that it'd be possible to trigger hacbs pipelines from appstudio workspace?

Reply:
backward compatibility, APIBinding cannot be used for non-kcp usage.

Reply:
Doesn't appstudio require kcp?

Reply:
Based on ASC-165 there should be still possible to run it on OpenShift.